lib/private/Config.php

```diff
@@ -215,12 +215,16 @@ private function readData() {
 
 		// Include file and merge config
 		foreach ($configFiles as $file) {
-			$fileExistsAndIsReadable = file_exists($file) && is_readable($file);
-			$filePointer = $fileExistsAndIsReadable ? fopen($file, 'r') : false;
-			if ($file === $this->configFilePath &&
-				$filePointer === false) {
-				// Opening the main config might not be possible, e.g. if the wrong
-				// permissions are set (likely on a new installation)
+			unset($CONFIG);
+
+			// Invalidate opcache (only if the timestamp changed)
+			if (function_exists('opcache_invalidate')) {
+				@opcache_invalidate($file, false);
```

Comment:
```suggestion
				opcache_invalidate($file, false);
```
Any reason for the `@` operator here?
I do not see any warning mention on https://www.php.net/manual/en/function.opcache-invalidate.php

Reply:
It was used in `writeData` below. Removed the `@`.